modules/iam-role-for-service-accounts-eks/policies.tf

```diff
@@ -628,6 +628,7 @@ data "aws_iam_policy_document" "fsx_lustre_csi" {
       "fsx:DeleteFileSystem",
       "fsx:DescribeFileSystems",
       "fsx:TagResource",
+      "fsx:UpdateFileSystem"
```

Comment:
we'll need to see this reflected in the upstream project's defined policy first before we add it here https://github.com/kubernetes-sigs/aws-fsx-csi-driver/blob/master/docs/install.md#set-up-driver-permissions